src/windows/main_window.py

```diff
@@ -2294,12 +2294,23 @@ def load_recent_menu(self):
             self.recent_menu.clear()
 
         # Add recent projects to menu
+        # Show just a placeholder menu, if we have no recent projects list
+        if not recent_projects:
+            self.recent_menu.addAction(_("No Recent Projects"))
```

Comment:
I was also thinking that this should be a _disabled_ menu item, maybe? (I have to remember how to do that...)

Reply:
I've added no handler for this action, so it just does nothing when clicked. We could disable it also, but I'm not sure it matters, IMO. :smile:

Reply:
I tried it, it looks a bit cleaner disabled IMHO. It's a visual thing, disabled it's _clearly_ not meant to be clicked.

Reply:
Thanks

Reply:
(Sounds like it has that user's name written all over it, too.) :wink: